### DISCUSSION TITLE
Release v20200506

### DISCUSSION THREAD

**Discussion by kevgliss** [Netflix/dispatch#1145] [General]
This is the first release of may and it's a big one!

With this release, we are now able to offer individual teams their own space to run their incidents. Per-project configuration of incident types, incident priorities, tags, notifications, plugins, etc, allows for multiple teams to use Dispatch in their own way. 

This is a big change that requires extensive database schema manipulation and data migration. As always, before upgrading. backup your existing data!

Features:
- Adds organizations/projects for incident resources (#990, #1010, #1011, #1019, #1023, #1024, #1028, #1035, #1036, #1040, #1047, #1050)
- Adds the ability to select FAIR Form of Loss category for incident types (#979)
- Adds a helpful direct message to the new incident commander (#1046)
- Adds the ability to scope tags by tag type while filtering (e.g. "applications/" will list all tags with tag-type of "applications") (#1067)
- Adds the ability to bulk delete incidents via web UI (#1069)
- Adds the ability to filter incidents by their tags (#1106)

Improvements:
- Consolidates restricted visibility-based filtering (#995)
- Moved from ace to monaco code editor (#1012)
- Makes company field optional for individual and team in Web UI (#1064)
- Unified web UI selects/combobox to include "load more" option (#1062, #1076)
- Creating a new search filter will automatically add it to the focused notification (#1066)
- Adds default column to incident priority table (#1090)
- Adds more response messages to slack modals and commands (#1132)
- Converts engagement filters to search filters (#1070)
- Updates individual contact info if it exists (#1129)
- Lots and lots of dependency updates

Bug Fixes:
- Fixes bug where the incident role was not being displayed correctly when changed via the Web UI (#1002)
- Fixes missing validation prop that prevented new individuals from being created in the UI (#1032)
- Fixes issue for incident reporter being incorrectly reported via API (#1055)
- Fixes issue where reporter/commander could not be changed if there was no incident conversation (#1054)

Documentation:
- Reworked a large amount of the documentation (#1088)

Security
- Fixes URL sanitization to prevent XSS (#1126)

<hr /><em>This discussion was created from the release <a href='https://github.com/Netflix/dispatch/releases/tag/v20200506'>Release v20200506</a>.</em>

**Comment by garmstrong67** [comment]
It seems with this update the documentation for configuring the plugins has disappeared, are there plans to update and add this back?  I had just started to work with the plugins and integrations, and now I'm at a bit of a loss.  Thanks!

**Reply by kevgliss** [reply]
[This](https://hawkins.gitbook.io/dispatch/contributing/plugins) is what we currently have for plugin documentation. I don't believe it's changed much... were you looking for anything specific?

Happy, to fill in any gaps in the documentation as well.

**Reply by garmstrong67** [reply]
Thanks for the response.

These are the pages I'm referring to in particular: https://hawkins.gitbook.io/dispatch/administration-guide/incident/plugins

For example: https://hawkins.gitbook.io/dispatch/administration-guide/incident/plugins/configuring-slack

These used to have information regarding how to configure these plugins using the env variables

**Comment by mvilanova** [comment]
@garmstrong67 https://github.com/Netflix/dispatch/pull/1184 should fix the broken plugin links.

**Reply by garmstrong67** [reply]
Thank you!
